.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,16 +24,11 @@ one platform) before opening the pull request.  `make run-functional-tests` and
 instance profile allowing it access to AWS resources.  Running
 `make run-functional-tests` and `.\scripts\run-functional-tests.ps1` may incur
 charges to your AWS account; if you're unable or unwilling to run these tests
-in your own account, we can run the tests and provide test results.
+in your own account, we can run the tests and provide test results. Also, once
+you open the pull request, there will be 14 automatic test checks on the bottom
+of the pull request, please make sure they all pass before you merge it, you can
```

Comment:
nit but this sentences runs on, chop it into 2 sentences?
.. before you merge it. You can use bot/test...